ProjectTemplates/VisualStudio2010/ContentPipelineExtension/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("$safeprojectname$")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Hewlett-Packard")]
+[assembly: AssemblyProduct("$safeprojectname$")]
+[assembly: AssemblyCopyright("Copyright © Hewlett-Packard 2015")]
```

Comment:
Can you fix this file....  it should look more like this:

https://github.com/mono/MonoGame/blob/develop/ProjectTemplates/VisualStudio2010/Windows/Properties/AssemblyInfo.cs